REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.6.0-pre
 StatsBase
 Distributions
+PositiveFactorizations
```

Comment:
In order for Newton algorithm to work far from a local minimum, when the hessian may not be positive defined, it's good to have an approximation of the inverse hessian that is positive defined, to ensure that the value of the objective function goes down and `PositiveFactorizations` is a small package that does just that: I use it [here](https://github.com/ararslan/Survival.jl/pull/2/files#diff-deac6c8d08fa6876d2f35a8bf1eb267dR7). Just using the inverse of the hessian would probably still work but in a less reliable way. Optim also has it as a dependency for the same reason I guess.